src/helpers.ts

```diff
@@ -45,6 +46,8 @@ export function findDuration(hass: HomeAssistant, config: TimerBarConfig, stateO
 /** Calculate the most accurate estimate of time remaining for the timer. */
 export const timerTimeRemaining = (hass: HomeAssistant, config: TimerBarConfig, stateObj: HassEntity): undefined | number => {
   const madeActive = new Date(stateObj.last_changed).getTime();
+  const duration = durationAttr(hass, stateObj, config.duration);
+  const remain_time = durationAttr(hass, stateObj, config.remain_time);
```

Comment:
It would be nice to rename this function as it's no longer being used for just duration. Maybe `getAttrFromConfig` or something?